app/controllers/photos_controller.rb

```diff
@@ -76,4 +72,6 @@ def set_photo
   def photo_params
     params.require(:photo).permit(:image, :comments_count, :likes_count, :caption, :owner_id)
```

Comment:
The use of strong parameters in photo_params helps prevent mass assignment vulnerabilities. However, including :owner_id in the permitted parameters might introduce a security risk, as it could allow users to set or change the owner of a photo through the request parameters.